rag/rag.py

```diff
@@ -18,16 +17,21 @@
 
 client = chromadb.PersistentClient(path="db/")
 print(f"{client.list_collections()}")
-collection_name = "test_embeddings"
 
 def retrieve(question):
     print(f'initial question: {question}')
-    collection = client.get_collection(collection_name)
-    qr = collection.query(query_texts=question, n_results=5)
-    ids = qr['ids'][0]
-    documents = qr['documents'][0]
-    # change this into a dict or something
-    res = {k: v for k, v in zip(ids, documents)}
+    # query_text = get_chat_completion(QUERY_SYSTEM_PROMPT, question)
+    # print(f'query text: {query_text}')
+    res={}
+    for document_type in ['shot', 'run']:
```

Comment:
So here we load `run_embeddings` and `shot_embeddings`. Meanwhile, in `embed_text.py`, we have a `text_embeddings`. Wondering if we need to add `text` here as well?